scripts/bump-version.bash

```diff
@@ -3,7 +3,7 @@
 set -e
 
 NEW_VERSION_NAME=$1
```

Comment:
awk now needs to print the 3rd element as there is now an equals sign in the middle for `versionName` and `versionCode`, like:
```
        versionCode = 4372
        versionName = "1.27.0-rc.2"
```

Reply:
Bumped to version `1.27.2-rc.2` now